<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BehaviourSettingsPage</class>
 <widget class="QWidget" name="BehaviourSettingsPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>915</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Behavior</string>
  </property>
  <layout class="QVBoxLayout" name="layout_behavioursettingspage">
   <item>
    <widget class="QCheckBox" name="checkbox_showtrayicon">
     <property name="text">
      <string>Show system tray icon</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_keeprunning">
     <property name="text">
      <string>Keep running in the background when the window is closed</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_trayicon_progress">
     <property name="text">
      <string>Show song progress on system tray icon</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_resumeplayback">
     <property name="text">
      <string>Resume playback on start</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_playingwidget">
     <property name="text">
      <string>Show playing widget</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="checkbox_artistbio">
     <property name="text">
      <string>Artist biography</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_startup">
     <property name="title">
      <string>On startup</string>
     </property>
     <layout class="QVBoxLayout" name="layout_startup">
      <item>
       <widget class="QRadioButton" name="radiobutton_remember">
        <property name="text">
         <string>Remember from &amp;last time</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_show">
        <property name="text">
         <string>Show the main window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_hide">
        <property name="text">
         <string>Hide the main window</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_show_maximized">
        <property name="text">
         <string>Show the main window maximized</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="radiobutton_show_minimized">
        <property name="text">
         <string>Show the main window minimized</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_language">
     <property name="title">
      <string>Language</string>
     </property>
     <layout class="QVBoxLayout" name="layout_language">
      <item>
       <widget class="QComboBox" name="combobox_language">
        <item>
         <property name="text">
          <string>Use the system default</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_language">
        <property name="text">
         <string>You will need to restart Strawberry if you change the language.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_add_song">
     <property name="title">
      <string>Using the menu to add a song will...</string>
     </property>
     <layout class="QVBoxLayout" name="layout_menuplaymode">
      <item>
       <widget class="QComboBox" name="combobox_menuplaymode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Never start playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Play if there is nothing already playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always start playing</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_previous">
     <property name="title">
      <string>Pressing &quot;Previous&quot; in player will...</string>
     </property>
     <layout class="QVBoxLayout" name="layout_previous">
      <item>
       <widget class="QComboBox" name="combobox_previousmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Jump to previous song right away</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Restart song, then jump to previous if pressed again</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_doubleclickaddmode">
     <property name="title">
      <string>Double clicking a song will...</string>
     </property>
     <layout class="QHBoxLayout" name="layout_doubleclickaddmode">
      <item>
       <widget class="QComboBox" name="combobox_doubleclickaddmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Append to the playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Replace the playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Open in new playlist</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add to the queue</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combobox_doubleclickplaymode">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <item>
         <property name="text">
          <string>Never start playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Play if there is nothing already playing</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Always start playing</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_doubleclickplaylist">
     <property name="title">
      <string>Double clicking a song in the playlist will...</string>
     </property>
     <layout class="QVBoxLayout" name="layout_doubleclickplaylist">
      <item>
       <widget class="QComboBox" name="combobox_doubleclickplaylistaddmode">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <item>
         <property name="text">
          <string>Change the currently playing song</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Add to the queue</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupbox_seekstep">
     <property name="title">
      <string>Seeking using a keyboard shortcut or mouse wheel</string>
     </property>
     <layout class="QHBoxLayout" name="layout_seekstep">
      <item>
       <widget class="QLabel" name="label_seekstep">
        <property name="text">
         <string>Time step</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinbox_seekstepsec">
        <property name="suffix">
         <string> s</string>
        </property>
        <property name="prefix">
         <string/>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>120</number>
        </property>
        <property name="value">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="spacer_seekstep">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer_bottom">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>5</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>checkbox_showtrayicon</tabstop>
  <tabstop>checkbox_keeprunning</tabstop>
  <tabstop>checkbox_trayicon_progress</tabstop>
  <tabstop>checkbox_resumeplayback</tabstop>
  <tabstop>checkbox_playingwidget</tabstop>
  <tabstop>radiobutton_remember</tabstop>
  <tabstop>radiobutton_show</tabstop>
  <tabstop>radiobutton_hide</tabstop>
  <tabstop>radiobutton_show_maximized</tabstop>
  <tabstop>radiobutton_show_minimized</tabstop>
  <tabstop>combobox_language</tabstop>
  <tabstop>combobox_menuplaymode</tabstop>
  <tabstop>combobox_previousmode</tabstop>
  <tabstop>combobox_doubleclickaddmode</tabstop>
  <tabstop>combobox_doubleclickplaymode</tabstop>
  <tabstop>combobox_doubleclickplaylistaddmode</tabstop>
  <tabstop>spinbox_seekstepsec</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
